### PR TITLE
Support for GlucoRx meters (UPLOAD-25)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses [UPLOAD-25], with support for:

- GlucoRx Nexus,
- GlucoRx Nexus Mini Ultra,
- GlucoRx Go,
- GlucoRx HCT.

[UPLOAD-25]: https://tidepool.atlassian.net/browse/UPLOAD-25?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/drivers/glucorx/glucoRxDriver.js [hunk 1]
```diff
@@ -0,0 +1,632 @@
...
+        delete postRecord.index;
+        data.post_records.push(postRecord);
+      }
+    });
+  };
+
+  const buildKetoneRecords = (data) => {
+    _.forEach(data.records, (record, index) => {
+      if (hasFlag(FLAGS.KETONE, record.flags)) {
+        let value = record.value / 30.0;
```
Comment (krystophv) [inline lib/drivers/glucorx/glucoRxDriver.js:403]: Is this magic `30.0` constant just a part of the spec?

**Review by krystophv — APPROVED** [review]
LGTM 🚀 

I did have one question about a sort of magic constant, but it doesn't hold up approval.

File: lib/drivers/glucorx/glucoRxDriver.js (hunk 1 above)
Comment (gniezen) [reply lib/drivers/glucorx/glucoRxDriver.js:403]: Yep, for some reason they decided to be able to store mmol/L values as integers they just multiply the value by 30 :shrug:

**Review by krystophv — APPROVED** [review]
Yup, updates look good 👍🏼

**Pull request merged**